doc/devel/min_dep_policy.rst

```diff
@@ -4,34 +4,56 @@
 Minimum Version of Dependencies Policy
 ======================================
 
+For the purpose of this document, 'minor version' is in the sense of
+SemVer (major, minor, patch) and includes both major and minor
+releases. For projects that use date-based versioning, every release
+is a 'minor version'.
+
+
 Python and numpy
 ================
 
-- support minor versions of ``Python`` initially released in the previous
-  3 years
-- support minor versions of ``numpy`` initially released in the
-  previous 3 years or oldest that supports the minimum python version
-  (which ever is higher)
+- support minor versions of ``Python`` initially released
```

Comment:
This should have 'in the' like the NumPy line, no? Right now it reads as if only Python older than 3 years is supported and nothing newer.